### PR TITLE
CFE#561:  Add additionalTrustBundlePolicy field

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/CFE-561

Link to docs preview:
https://50442--docspreview.netlify.app/openshift-enterprise/latest/networking/configuring-a-custom-pki.html#installation-configure-proxy_configuring-a-custom-pki

### CODE REVIEW

File: modules/installation-configure-proxy.adoc [hunk 1]
```diff
@@ -218,7 +219,8 @@ endif::aws[]
 ifdef::vsphere,vmc[]
 You must include vCenter's IP address and the IP range that you use for its machines.
 endif::vsphere,vmc[]
-<4> If provided, the installation program generates a config map that is named `user-ca-bundle` in
+<4> Optional: The policy to add the `AdditionalTrustBundle` to the trusted certificate store of nodes. The allowed values are `Proxyonly` and `Always`. Use `Proxyonly` to add the `AdditionalTrustBundle` only when `http/https` proxy is configured, and `Always` to always add the `AdditionalTrustBundle``. The default value is `Proxyonly`.
```
Comment (TrilokGeer) [inline modules/installation-configure-proxy.adoc:222]: Should there be a "If provided" statement for policy to describe the behaviour? Like "If provided, the `Proxy` object is configured to reference the `user-ca-bundle` config map in the `trustedCA` field"

Comment (snarayan-redhat) [reply modules/installation-configure-proxy.adoc:222]: Here `If provided` = `Optional:` But I can replace the existing `add the `AdditionalTrustBundle` to the trusted certificate store` to what you have suggested. Any thoughts on this?

Comment (TrilokGeer) [reply modules/installation-configure-proxy.adoc:222]: Sounds good to me, thanks

Comment (snarayan-redhat) [reply modules/installation-configure-proxy.adoc]: I have incorporated your suggestions as well as changed the sequence of `additionalTrustBundlePolicy` as discussed

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

File: modules/installation-configure-proxy.adoc [hunk 2]
```diff
@@ -225,6 +226,7 @@ Operator then creates a `trusted-ca-bundle` config map that merges the contents
 with the {op-system} trust bundle. The `additionalTrustBundle` field is required unless
 the proxy's identity certificate is signed by an authority from the {op-system} trust
 bundle.
+<5> Optional: The policy to determine the configuration of the `Proxy` object to reference the `user-ca-bundle` config map in the `trustedCA` field. The allowed values are `Proxyonly` and `Always`. Use `Proxyonly` to reference the `user-ca-bundle` config map only when `http/https` proxy is configured, and `Always` to always reference the `user-ca-bundle` config map. The default value is `Proxyonly`.
```
Comment (michaelryanpeter) [inline modules/installation-configure-proxy.adoc:229]:
```suggestion
<5> Optional: The policy to determine the configuration of the `Proxy` object to reference the `user-ca-bundle` config map in the `trustedCA` field. The allowed values are `Proxyonly` and `Always`.  The default value is `Proxyonly`. Use `Proxyonly` to reference the `user-ca-bundle` config map only when `http/https` proxy is configured. Use `Always` to always reference the `user-ca-bundle` config map.
```
This is just a minor suggestion. You have done a good job using the principles of minimalism here. I think that by breaking up the long sentence, using parallel structure, and moving the `Proxyonly` information closer together, it is a little bit easier to read. WDYT?

**Pull request merged**